x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/SecurityTests.java

```diff
@@ -686,10 +686,10 @@ public void testSecurityRestHandlerWrapperCanBeInstalled() throws IllegalAccessE
             // We will throw later if another wrapper is already installed
```

Comment:
Nit:
```suggestion
            // We will throw later if another interceptor is already installed
```

I cannot comment on the line above this one. But it also uses the word `wrapper` instead of `interceptor`. The same goes for the name of this method and the previous one.

Reply:
Good catch. Fixed.